src/frontend/src/utils/status.ts

```diff
@@ -174,6 +176,9 @@ export const statusList = (pia: IPiaForm | null): StatusList => {
       Pages: {
         review: {
           accessControl: false,
+          params: {
```

Comment:
note: if accessControl is false this means in this status this page should not be shown to anyone.